### DISCUSSION TITLE
How to config Spark to use spline using confinguration file

### DISCUSSION THREAD

**Discussion by zacayd** [AbsaOSS/spline-spark-agent#852] [Q&A]
Hi
How can i configure spark to use spline  but no on a promt 
**not like**

```
spark-shell \
  --packages za.co.absa.spline.agent.spark:spark-3.3-spline-agent-bundle_2.12:2.2.0 \
  --conf "spark.sql.queryExecutionListeners=za.co.absa.spline.harvester.listener.SplineQueryExecutionListener" \
  --conf "spark.spline.lineageDispatcher.http.producer.url=http://10.0.19.4:8080/producer"  \
  --conf "spark.driver.extraJavaOptions=--add-opens java.base/sun.net.www.protocol.jar=ALL-UNNAMED" \
  --conf "spark.executor.extraJavaOptions=--add-opens java.base/sun.net.www.protocol.jar=ALL-UNNAMED"
```

I mean that once I  put on config files of spark and from there i can run scala and py files and see lineage
thanks
Zacay

**Comment by wajda** [comment]
It depends on your Spark distribution, where to put the configuration so that it's applied globally, and where to put the Spline JAR-file, so that it's included in the classpath of the Spark JVM.
What Spark distr are you using?
